source/index.html.md

```diff
@@ -897,3 +958,89 @@ test_type | - | The ***test_type*** of the ***Covid test***.
 test_result | - | The ***test_result*** of the ***Covid test***.
 issuer_email | - | The ***issuer_email*** of the ***Covid test*** records the email address of the issuer for accountability and auditing purposes.
 issuer_location | - | The ***issuer_location*** of the ***Covid test*** records the location where the test was performed for accountability and auditing purposes.
+
+## Verify the Signature of a Covid-free Certificate
+```python
+import requests
+data = {
+    'api_secret': '676bc3ca-e2b9-4161-85a4-1dc8592916a5',
```

Comment:
@robin-ankele Do we want to keep the api_secret public?

Reply:
@ratuliut this is just a random uuid to show what's the format

Reply:
Got it - thanks! :-)